Dockerfile.pr-builder

```diff
@@ -29,46 +30,58 @@ RUN apt-get update && apt-get -y install \
 RUN Rscript -e 'install.packages(c("jsonlite", "SKAT", "logistf"), repos = "http://cran.us.r-project.org")' && \
     Rscript -e 'source("https://bioconductor.org/biocLite.R"); biocLite("GENESIS"); biocLite("SNPRelate"); biocLite("GWASTools")'
 
+# this seems easier than getting the keys right for apt
+#
+# source: https://cloud.google.com/storage/docs/gsutil_install#linux
+RUN /bin/sh -c 'curl https://sdk.cloud.google.com | bash' && \
+    mv /root/google-cloud-sdk /
+ENV PATH $PATH:/google-cloud-sdk/bin
+
+RUN wget -O  /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.11.3/bin/linux/amd64/kubectl && \
+    chmod +x /usr/local/bin/kubectl
+
+WORKDIR /spark
+RUN wget -O spark-2.2.0-bin-hadoop2.7.tgz https://archive.apache.org/dist/spark/spark-2.2.0/spark-2.2.0-bin-hadoop2.7.tgz && \
+    tar --strip-components=1 -xvzf spark-2.2.0-bin-hadoop2.7.tgz -C /spark && \
+    rm -rf spark-2.2.2-bin-hadoop2.7.tgz
```

Comment:
Shouldn't this either download spark 2.2.2, or remove `spark-2.2.0-bin-hadoop2.7.tgz`?